SoftLayer/managers/account.py

```diff
@@ -303,3 +303,25 @@ def get_network_message_delivery_accounts(self):
         _mask = """vendor,type"""
 
         return self.client['SoftLayer_Account'].getNetworkMessageDeliveryAccounts(mask=_mask)
+
+    def get_active_virtual_licenses(self):
+        """Gets all active virtual licenses account.
+
+                :returns: active virtual licenses account
+                """
```

Comment:
Indentation should be the same as the start of this comment block 
```suggestion
        """Gets all active virtual licenses account.

        :returns: active virtual licenses account
        """
```